### PR TITLE
feat:add card component

### PR DESCRIPTION
**Pull request by shubhamkumar648**
add variant of code:
1.text only card
2.vertical card
card capture

3.horizontal card
4.card with badges
5 card with shadow

### CODE REVIEW

File: styles/card.css [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    gap: 1rem;
+}
+
+.card_body {
+    border: 1px solid var(--primary-color);
+}
+
+.card {
+    width: 15rem;
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
```
Comment (rahulcs754) [inline styles/card.css:13]: try to color variable

File: styles/card.css [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+.card_badge_right {
+    top: 5px;
+    right: 5px;
+    position: absolute;
+    color: var(--lightwhite-color);
+    background-color: var(--primary-color);
+    border-radius: 1px solid (--lightwhite-color);
+    padding: 3px 5px;
+
```
Comment (rahulcs754) [inline styles/card.css]: remove space

**Review by rahulcs754 — COMMENTED** [review]
I checked your code. here are some changes.
I think you should create some  card height class.

**Review by ashishnnnnn — APPROVED** [review]
Code seems good :+1:

File: styles/card.css (hunk 1 above)
Comment (shubhamkumar648) [reply styles/card.css:13]: thank you for aknowledge ! done with latest commit

**Review by rahulcs754 — APPROVED** [review]

File: styles/card.css (hunk 2 above)
Comment (shubhamkumar648) [reply styles/card.css]: done

**Pull request merged**